src/components/Navbar.tsx

```diff
@@ -8,17 +14,13 @@ export const Navbar = () => {
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink className={isActiveLink} to="/">
```

Comment:
The `NavLink` component has a built-in `activeClassName` prop that you can use to style the active link. Instead of using a custom function `isActiveLink`, consider using the `activeClassName` prop. For example, you can write: `<NavLink activeClassName="has-background-grey-lighter" to="/">`.